src/commands/profile.ts

```diff
@@ -0,0 +1,34 @@
+import { ApplicationCommandOptionType } from 'discord.js';
+
+export const profile: GlobalCommand = {
+	name: 'profile',
+	description: 'Responds with the Profile Picture of the mentioned user',
+	options: [
+		{
+			name: 'user',
+			description: 'The user to get the profile picture of',
+			type: ApplicationCommandOptionType?.User,
+		},
+	],
+	requiresGuild: false,
+	async execute({ interaction, options }) {
+		const user = interaction.user;
+		const param = options[0];
+		if (param?.value === undefined) {
+			await interaction.reply({
+				content: user.avatarURL({ extension: 'png', size: 2048 }) as string,
+				ephemeral: true,
+			});
+		} else if (param?.user?.avatar !== null) {
+			await interaction.reply({
+				content: param?.user?.avatarURL({ extension: 'png', size: 2048 }) as string,
```

Comment:
Since the previous condition handled `param` being `undefined`, these two cases don't need to coalesce.
```suggestion
		} else if (param.user?.avatar !== null) {
			await interaction.reply({
				content: param.user?.avatarURL({ extension: 'png', size: 2048 }) as string,
```

Reply:
We should maybe also handle this `null` case, rather than force-casting it away.